core/src/test/java/io/questdb/cutlass/http/ImportIODispatcherTest.java

```diff
@@ -184,7 +156,8 @@
             "  {\r\n" +
             "    \"name\": \"Pickup_DateTime\",\r\n" +
             "    \"type\": \"TIMESTAMP\",\r\n" +
-            "    \"pattern\": \"yyyy-MM-dd HH:mm:ss\"\r\n" +
+            "    \"pattern\": \"yyyy-MM-dd HH:mm:ss\",\r\n" +
+            "    \"designated\": \"true\"\r\n" +
```

Comment:
There is already way to mark timestamp designated in as URL get parameter. 
See https://questdb.io/docs/reference/api/rest/#imp---import-data

```
curl \
-F schema='[{"name":"ts", "type": "TIMESTAMP", "pattern": "yyyy-MM-dd - HH:mm:ss"}]' \
-F data=@weather.csv \
'http://localhost:9000/imp?overwrite=true&timestamp=ts&partitionBy=MONTH'
```

*timestamp=ts*

Reply:
indeed, I revert the changes, except I keep a test as a reminder